plugins/module_utils/network/sonic/config/port_breakout/port_breakout.py

```diff
@@ -175,6 +176,56 @@ def _state_deleted(self, want, have, diff):
 
         return commands, requests
 
+    def _state_replaced(self, want, have, diff):
+        """ The command generator when state is replaced
+
+        :param want: the additive configuration as a dictionary
+        :param obj_in_have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+
+        commands = diff
+        requests = self.get_modify_port_breakout_requests(commands, want)
+        if commands and len(requests) > 0:
+            commands = update_states(commands, "replaced")
+        else:
+            commands = []
+
+        return commands, requests
+
+    def _state_overridden(self, want, have, diff):
+        """ The command generator when state is overridden
+
+        :param want: the additive configuration as a dictionary
+        :param obj_in_have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        commands = []
+        requests = []
+        if not diff:
+            return commands, requests
+        commands = have
+        requests = self.get_delete_port_breakout_requests(commands, have)
+
+        if commands and len(requests) > 0:
+            send_requests(self._module, requests)
+        else:
+            commands = []
+
+        commands = diff
```

Comment:
This will work if the modified deletion logic implemented to address the previous comment avoids deleting breakout modes that already match the requested new mode in "want". If the modified logic does a "delete all", then commands should be set equal to 'want' here.